app/assets/stylesheets/active_admin/components/_popovers.scss

```diff
@@ -0,0 +1,119 @@
+.popover {
+  display: inline-block;
+  position: absolute;
+  background-color: white;
+  padding: 2px;
+  box-shadow: rgba(0,0,0,0.4) 0 1px 3px, lighten($primary-color, 15%) 0px 1px 0px 0px inset;
+  background-color: $primary-color;
+  border: solid 1px darken($primary-color, 10%);
+  border-top-color: darken($primary-color, 4%);
+  border-bottom-color: darken($primary-color, 17%);
+
+  border-radius: 4px;
+
+  .popover_nipple{
+
+    // The nipple's border
+    content: "";
```

Comment:
Properties should be ordered border-color, border-style, border-width, content, display, height, position, top, width, z-index